### PR TITLE
Update Hosting documentation 

### PR DESCRIPTION
**Pull request by FirelightFlagboy**
- Rename adminguide to hosting in the documentation
- Add hosting introduction section
- Rename hosting to deployment
- Update deployment preamble
- Rename `migrate` section to `maintenance` in hosting guide
- Move administration docs to hosting/administration
- Rename `hosting/installation` to `hosting/install_cli`

### CODE REVIEW

File: docs/hosting/deployment/index.rst [hunk 1]
```diff
@@ -2,31 +2,31 @@
...
 
-==========================
-Parsec Server installation
-==========================
+=================
+Server Deployment
+=================
 
-This guide covers the installation procedure for the `Parsec`_ server.
+This guide covers the deployment procedure for the `Parsec`_ server.
```
Comment (mmmarcos) [inline docs/hosting/deployment/index.rst:17]: I'm not sure the link to https://parsec.cloud is that useful here
```suggestion
This section covers how to deploy the Parsec server .
```

File: docs/hosting/introduction.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.. _doc_hosting_intro:
+
+Hosting Introduction
+====================
```
Comment (mmmarcos) [inline docs/hosting/introduction.rst:4]:
```suggestion
Introduction
============
```

File: docs/hosting/introduction.rst [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+This section is dedicated to the hosting of Parsec. It will guide you through the deployment, maintenance and administration of the Parsec server and its components.
```
Comment (mmmarcos) [inline docs/hosting/introduction.rst:6]: I'll keep it short here
```suggestion
This guide describes Parsec server deployment, maintenance and administration.
```
Or maybe a slightly more "friendly":
```suggestion
This guide will help you to deploy, maintain and manage your own Parsec server.
```

File: docs/hosting/maintenance/index.rst [hunk 4]
```diff
@@ -1,31 +1,31 @@
...
 
-=======================
-Parsec Server migration
-=======================
+==================
+Server Maintenance
+==================
 
-This guide covers the migration procedure for the `Parsec`_ server (also called `parsec-server`).
+This guide covers the maintenance procedure for the `Parsec`_ server (also called `parsec-server`).
```
Comment (mmmarcos) [inline docs/hosting/maintenance/index.rst:11]:
```suggestion
This section covers how to maintain the Parsec server up to date.
```

File: newsfragments/7844.doc.rst [hunk 5]
```diff
@@ -0,0 +1,4 @@
+Update hosting documentation:
+
+Previously know as the section **Administration** it has been renamed to **Hosting**.
+It's subsection **Hosting** as been renamed to **Deployment**.
```
Comment (mmmarcos) [inline newsfragments/7844.doc.rst:4]: I wouldn't add much detail here about renamed or moved sections. 
For me, as a user, the important thing is that the guide is has been updated to be consistent with V3.
```suggestion
Update hosting guide (previously administration guide) for version 3
```

**Review by mmmarcos — COMMENTED** [review]
A couple of comment, otherwise hosting guide looks good!

File: docs/hosting/deployment/index.rst [hunk 6]
```diff
@@ -2,31 +2,31 @@
...
 
-==========================
-Parsec Server installation
-==========================
+=================
+Server Deployment
+=================
 
-This guide covers the installation procedure for the `Parsec`_ server.
+This section covers how to deploy the Parsec server .
```
Comment (mmmarcos) [inline docs/hosting/deployment/index.rst:17]:
```suggestion
This section covers how to deploy the Parsec server.
```

**Review by mmmarcos — APPROVED** [review]
LGTM :+1:

**Review by AureliaDolo — APPROVED** [review]

**Pull request merged**